src/astarte-client/models/Token/index.ts

```diff
@@ -139,4 +139,8 @@ export class AstarteToken {
       return 'invalid';
     }
   }
+
+  getClaims(): Claims {
+    return this.$claims;
+  }
```

Comment:
I suggest using the already implemented `can` function instead of exposing the claims.

For example, to verify authorization to access the Devices page I could check that:

```ts
astarteToken.can("appEngine", "GET", "/devices")
```

The third argument, the resource, refers to the API path as described in the API Swagger files:

https://docs.astarte-platform.org/astarte/snapshot/api/index.html?urls.primaryName=AppEngine%20API#/

Note that for simplicity in my example I'm matching just a portion of the API path, without the `/{realm_name}` part.